plugins/onepad/joystick.cpp

```diff
@@ -218,7 +218,7 @@ bool JoystickInfo::Init(int id)
 	// Keep the 4 hat buttons too (usb driver). (left pressure does not work with recent kernel). Moreover the pressure
 	// work sometime on half axis neg others time in fulll axis. So better keep them as button for the moment
 	u32 found_hack = devname.find(string("PLAYSTATION(R)3"));
-	if (found_hack != string::npos) {
+	if (found_hack != string::npos && (conf->options & PADOPTION_SIXAXIS_USB) == 0) {
```

Comment:
The desired default behavior is to use the axis events instead of the toggle events to leverage the pressure sensitivity.  With the box checked the full range of toggle events is processed (my patch) and the axis events for the Dpad are ignored (original) code.  Recent versions of evdev don't report the Dpad axes to SDL anyway, so the legacy code does nothing.  The only place where my patch might cause trouble is if someone were using an old linux and evdev version.  Then with the box checked OnePad would process toggle and axis events for all the pressure sensitive buttons except the Dpad.